package.json

```diff
@@ -5,7 +5,7 @@
   "main": "build/index.js",
   "scripts": {
     "build": "rm -rf ./build && tsc",
-    "test": "jest",
```

Comment:
Yes, if it makes sense, right now we have e2e, so that gives us some coverage, but with these changes these tests started to fail and it wasn't clear at first how I can mock the new parts effectively, and also it didn't really test anything, it just basically tested if the dummy values are equal with the dummy values